documentation/src/test/java/org/junit/api/tools/ApiReportGenerator.java

```diff
@@ -62,24 +63,23 @@ ApiReport generateReport(String... packages) {
 		final String EOL = System.lineSeparator();
 
 		// Scan packages
-		ScanResult scanResult = new FastClasspathScanner(packages).scan();
+		ScanResult scanResult = new ClassGraph().whitelistPackages(packages).enableAnnotationInfo().scan();
 
 		// Collect names
-		List<String> names = new ArrayList<>();
-		names.addAll(scanResult.getNamesOfClassesWithAnnotation(API.class));
-		names.addAll(scanResult.getNamesOfAnnotationsWithMetaAnnotation(API.class));
```

Comment:
I'm not sure if this change is correct (see: https://github.com/classgraph/classgraph/issues/215#issuecomment-408674710)

Reply:
`Annotation APIs now handle meta-annotations` (source: https://github.com/classgraph/classgraph/wiki/Porting-FastClasspathScanner-code-to-ClassGraph), so this change is fine.